global_planner/src/orientation_filter.cpp

```diff
@@ -97,6 +97,26 @@ void OrientationFilter::processPath(const geometry_msgs::PoseStamped& start,
                 }
             }
             break;
+        case SIDEWARD:
+            for (int i = 0; i < n - 1; i++) {
+                setAngleBasedOnPositionDerivative(path, i);
+            }
+            if (n > 2) {
+                const int num_skips = n >= 5 ? 2 : 1;
+                const double rotation_to_path =  min_angle(start, *std::next(path.begin(), num_skips));
+                const double rotation_to_goal =  min_angle(*std::prev(path.end(), 1 + num_skips), path.back());
```

Comment:
nit: maybe use the same variable names as in lines 89-91 (since it is the same code)?